app/models/vendor.rb

```diff
@@ -6,4 +6,8 @@ class Vendor < ActiveRecord::Base
   has_many :locations, through: :vendor_location_schedules
 
   validates_presence_of :vendor_name
+
+  def self.approve(vendor)
```

Comment:
If you're going to call `Vendor.approve(some_specific_vendor)`, it's always better to make it an instance method instead so you can call `some_specific_vendor.approve`